.github/workflows/docs.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy documentation
+
+on: [push]
+# on:
+#   push:
+#     branches:
+#       - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Build and push docs
+        run: |
+          git config user.name Packit
+          git config user.email user-cont-team@redhat.com
+
+          pip3 install --user pdoc3 pygments
+
+          pdoc --html -o docs ogr
+          touch .nojekyll
+
+          git checkout -b gh-pages
```

Comment:
Wait! Where is this going to be available? :)

Reply:
Would it be better (or even feasible) to open a PR against the main branch with the docs and serve them from main?

Reply:
I was trying it out on my repo: https://mfocko.github.io/ogr/index.html

I'm also going to change the action trigger to _push to main_ and enable GitHub pages after the merge